draft-ietf-quic-http.md

```diff
@@ -1385,8 +1385,9 @@ HTTP_VERSION_FALLBACK (0x09):
 HTTP_WRONG_STREAM (0x0A):
 : A frame was received on a stream where it is not permitted.
 
-HTTP_PUSH_LIMIT_EXCEEDED (0x0B):
-: A Push ID greater than the current maximum Push ID was referenced.
+HTTP_LIMIT_EXCEEDED (0x0B):
+: An identifier greater than the current maximum for that identifier was
+  referenced.
```

Comment:
I think that you can (and should) list the types of identifiers this applies to.  I assume that this is limited to push IDs and priority placeholders.

Reply:
And stream IDs.